pages/linux/firewall-cmd.md

```diff
@@ -10,6 +10,10 @@
 
 `firewall-cmd --list-all`
 
+- Permanently move the interface into the trusted zone effectively allowing all communication:
```

Comment:
```suggestion
- Permanently move the interface into the trusted zone, effectively allowing all communication:
```
Maybe this could be made even more clear.

Reply:
Sure.